src/main/java/se/kth/sda/skeleton/comments/Comment.java

```diff
@@ -21,6 +24,15 @@
     @NotBlank
     private String contentText;
 
+    @ManyToOne
+    @JsonIdentityInfo(generator = ObjectIdGenerators.PropertyGenerator.class, property = "name")
+    @JsonIdentityReference(alwaysAsId = true)
+    @JoinColumn(nullable = false)
+    private User relatedUser;
+    
+    @OneToMany(mappedBy = "likedPost", cascade = CascadeType.ALL)
+    private List<PostLike> listOfCommentLikes;
```

Comment:
Please remove this we will implement later

Reply:
it's been removed